src/screens/Episode/components/Player/Player.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react'
+import { View, Pressable } from 'react-native'
+import TrackPlayer, { useProgress, Capability } from 'react-native-track-player'
+import Slider from '@react-native-community/slider'
+import { FormattedEpisodeData } from 'src/screens/Episode/common'
+import { getTime } from 'src/screens/Episode/helpers'
+import { PlainText } from 'src/components'
+import PlayIcon from 'src/assets/images/play.svg'
+import PauseIcon from 'src/assets/images/pause.svg'
+import RewindIcon from 'src/assets/images/rewind.svg'
+import ForwardIcon from 'src/assets/images/forward.svg'
+import styles from './styles'
+
+type Props = {
+  episode: FormattedEpisodeData
+}
+
+const Player: React.FC<Props> = ({ episode }) => {
+  const { position, duration } = useProgress()
+  const [isPlaying, setPlaying] = useState(false)
+
+  const trackPlayerInit = async () => {
+    await TrackPlayer.setupPlayer({
+      maxCacheSize: 1048576
+    })
+    return true
+  }
+
+  useEffect(() => {
+    trackPlayerInit()
+    TrackPlayer.updateOptions({
+      stopWithApp: false,
+      alwaysPauseOnInterruption: true,
+      capabilities: [Capability.Play, Capability.Pause, Capability.Stop],
+      compactCapabilities: [Capability.Play, Capability.Pause, Capability.Stop],
+      notificationCapabilities: [
+        Capability.Play,
+        Capability.Pause,
+        Capability.Stop
+      ]
+    })
+    TrackPlayer.add(episode)
+  }, [])
+
+  const onPressFunction = () => {
+    TrackPlayer.play()
+    setPlaying(true)
+  }
+  const onPressPause = () => {
+    TrackPlayer.pause()
+    setPlaying(false)
+  }
+  const onHandleSeekTo = (seconds: number) => {
+    TrackPlayer.seekTo(seconds)
+  }
+
+  return (
+    <>
```

Comment:
I think, it will be better if we wrap it by simple `View` component

Reply:
I added additional View, but i'm not sure that it good idea, because it additional element to render.

Reply:
> I added additional View, but i'm not sure that it good idea, because it additional element to render.
But we should wrap 2 nested <View> components. And <> is not good idea for it.